community/contributing/code_style_guidelines.rst

```diff
@@ -301,3 +301,62 @@ Editor integration
 Many IDEs or code editors have beautifier plugins that can be configured to run
 black automatically, for example each time you save a file. For details you can
 check `Black editor integration <https://github.com/psf/black#editor-integration>`__.
+
+Comment style guide
+-------------------
+
+This comment style guide applies to all programming languages used within
+Godot's codebase.
+
+- Use sentence case for comments. Begin comments with an uppercase character and
+  always end them with a period.
+- Reference variable/function names and values using backticks.
+- Wrap comments to ~100 characters.
```

Comment:
This is just my opinion, I think we should allow up to 100 but if there's going to be wrapping we can do so to 80 characters.

```suggestion
- Wrap comments to 80 characters if the line would exceed 100 characters.
```

Reply:
I'm not sure about it. Wrapping to 80 characters in C++ can make for difficult-to-read comments if they're heavily indented.